### PR TITLE
Migrating Some Test Fixtures to Pester

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Test/Invoke-Rivet.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+Set-StrictMode -Version 'Latest'
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Test.ps1' -Resolve)
+
+function Assert-OperationsReturned
+{
+    param(
+        [object[]]
+        $Operation
+    )
+    ps
```
Comment (decoyjoe) [inline Test/Invoke-Rivet.Tests.ps1]: Errant string?

**Review by decoyjoe — DISMISSED** [review]

**Comment by splatteredbits** [comment]
@DecoyJoe Ready for next review.

File: Test/RivetTest/Assert-View.ps1 [hunk 2]
```diff
@@ -26,18 +26,33 @@ function Assert-View
...
-    
-    if( $PSBoundParameters.ContainsKey('Definition') )
-    {
-        $expectedDefinition = "create view [{0}].[{1}] {2}" -f $SchemaName, $Name, $Definition
-        Assert-Match $view.definition ([Text.RegularExpressions.Regex]::Escape($expectedDefinition))
-    }
 
-    if( $PSBoundParameters.ContainsKey('Description') )
+    $expectedDefinition = "create view [{0}].[{1}] {2}" -f $SchemaName, $Name, $Definition
+    if( (Test-Path -Path 'TestDrive:') )
```
Comment (decoyjoe) [inline Test/RivetTest/Assert-View.ps1]: This can be updated to `Test-Pester`

File: Test/RivetTest/Assert-StoredProcedure.ps1 [hunk 3]
```diff
@@ -18,19 +18,34 @@ function Assert-StoredProcedure
...
-    Assert-NotNull $sp ('Stored Procedure {0}.{1} doesn''t exist.' -f $SchemaName,$Name)
+    $sp = Get-StoredProcedure -SchemaName $SchemaName -Name $Name -DatabaseName $DatabaseName
+
+    $expectedDefinition = "CREATE procedure [{0}].[{1}] {2}" -f $SchemaName, $Name, $Definition
 
-    if( $PSBoundParameters.ContainsKey('Definition') )
-    {    
-        $expectedDefinition = "CREATE procedure [{0}].[{1}] {2}" -f $SchemaName, $Name, $Definition
-        Assert-Match $sp.definition ([Text.RegularExpressions.Regex]::Escape($expectedDefinition))
+    if( (Test-Path -Path 'TestDrive:') )
```
Comment (decoyjoe) [inline Test/RivetTest/Assert-StoredProcedure.ps1]: This can be updated to `Test-Pester`

File: Test/Invoke-MigrationOperation.Tests.ps1 [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+    {
+        Remove-Table 'Foobar'
+    }
+'@ | New-TestMigration -Name 'ValidateMigrations'
+    
+        try
+        {
+            Invoke-RTRivet -Push 'ValidateMigrations' -ErrorAction SilentlyContinue
+            (Test-Trigger -Name 'trFoobar_Nothing') | Should Be $false
+            $Global:Error.Count | Should BeGreaterThan 0
```
Comment (decoyjoe) [inline Test/Invoke-MigrationOperation.Tests.ps1]: Maybe this was intentional but the rest of this PR uses the Pester v4 switch syntax, e.g. `Should -BeGreaterThan 0`, but this whole file uses the v3 syntax, e.g. `Should BeGreaterThan 0`

**Review by decoyjoe — DISMISSED** [review]

**Review by decoyjoe — APPROVED** [review]

**Pull request merged**